platforms/core-configuration/kotlin-dsl/src/main/kotlin/org/gradle/kotlin/dsl/PropertyExtensions.kt

```diff
@@ -194,6 +194,45 @@ operator fun <K : Any, V : Any> MapProperty<K, V>.plusAssign(value: Pair<K, V>)
     this.put(value.first, value.second)
 }
 
+/**
+ * Returns a provider that resolves to the value of the mapping of the given key. It will have no value
+ * if the property has no value, or if it does not contain a mapping for the key.
+ *
+ * @see MapProperty.getting
+ * @since 9.0
+ */
+@Incubating
+operator fun <K : Any, V : Any> MapProperty<K, V>.get(key: K): Provider<V>? {
```

Comment:
❓ Do we return null provider if key is missing? Relates to the `?` in `Provider<V>?`

Reply:
Good catch, `MapProperty.getting()` never returns null